utils/luci-app-ffwizard-berlin/root/www/cgi-bin/has_root-pass

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
+echo "Content-type: text/plain"
+echo ""
+USER=root
+# taken from http://serverfault.com/a/240961
+grep ${USER} /etc/shadow | awk -F":" '{ if ($2 == "!" || $2 == "*" || $2 == "") \
```

Comment:
With a locked root-account will you be able to login to LUCI at all? 

But I can remove the "!" if you prefer.